Executive.cpp

```diff
@@ -0,0 +1,86 @@
+/*
+
+Humberto Dominguez
+Professor Scott Roueche
+CSE 687- Object Oriented Design
+Syracuse University
+Phase 1 - Executive cpp
+4/20/2023
+
+This class was created in part of creating the MapReduce code for Phase 1 for
+CSE 687. The Executive class's primary purpose is to gather all data related to directories.
+
+Since this is the cpp portion of the class, this will define how and where the the member and methods will be
+used.
+*/
+
+//libraries that will be used in this class
+#include "Executive.h"
+
+//default constructor 
+Executive::Executive() {}
+
+//default destructor 
+Executive::~Executive() {}
+
+//mathod that will obtain the directories and test to verify they exist

```

Comment:
Review some of your methods below, a good amount of these methods aren't necessary. The main objective is to create an object of workflow with input, output and intermediate directories as arguments and call the execute() on it.

Reply:
So should we just delete the get methods and just directly enter in the variables directly?